internal/pkg/cli/app_init.go

```diff
@@ -172,14 +172,26 @@ func (opts *AppInitOpts) askDockerfile() error {
 	return nil
 }
 
-// listDockerfiles returns the list of Dockerfiles within a sub-directory below current working directory.
+// listDockerfiles returns the list of Dockerfiles within the current working directory and a sub-directory level below.
 // If an error occurs while reading directories, returns the error.
 func (opts *AppInitOpts) listDockerfiles() ([]string, error) {
 	wdFiles, err := afero.ReadDir(opts.fs, ".")
 	if err != nil {
 		return nil, fmt.Errorf("read directory: %w", err)
 	}
 	var dockerfiles []string
+
+	// Add Dockerfiles in current working directory.
+	for _, f := range wdFiles {
```

Comment:
Can we avoid looping through `wdFiles` twice?

Maybe by replacing:
```go
if !wdFile.IsDir() {
  continue
}
```
with:
```go
if !wdFile.IsDir() && wdFile.Name() == "Dockerfile" {
  dockerfiles = append(dockerfiles, filepath.Join(".", wdFile.Name()))
  continue
}
```

Reply:
I did that on purpose thinking it was more readable since there is no performance concern but sure i can change it

Reply:
Also does that work if the file name is not "Dockerfile"?

Reply:
```go
if !wdFile.IsDir() {
  if wdFile.Name() == "Dockerfile" {
    dockerfiles = append(dockerfiles, filepath.Join(".", wdFile.Name()))
  }
  continue
}
```

Reply:
Yeah, I usually try to avoid nested if-statements because I find it more complex but this isn't too bad.